src/snowcli/cli/nativeapp/init.py

```diff
@@ -239,36 +281,70 @@ def _init_without_user_provided_template(
         with TemporaryDirectory(dir=current_working_directory) as temp_dir:
             # Checkout the basic template, which will now reside at ./native-apps-templates/native-app-basic
             _sparse_checkout(
-                git_url=SNOWFLAKELABS_GITHUB_URL,
-                repo_sub_directory=BASIC_TEMPLATE,
+                git_url=git_url,
+                repo_sub_directory=template,
                 target_parent_directory=temp_dir,
             )
 
             # Move native-app-basic to current_working_directory and rename to name
             _move_and_rename_project(
                 source_parent_directory=current_working_directory.joinpath(temp_dir),
                 target_parent_directory=current_working_directory,
-                repo_sub_directory=BASIC_TEMPLATE,
+                repo_sub_directory=template,
                 new_name=project_name,
             )
 
-        # Render snowflake.yml file from its jinja template
-        render_snowflake_yml(
-            parent_to_snowflake_yml=current_working_directory.joinpath(project_name)
-        )
+        # If cloning the basic template from Snowflake official templates repo, render jinja templates to corresponding files.
+        if git_url == SNOWFLAKELABS_GITHUB_URL and template == BASIC_TEMPLATE:
+            # Render snowflake.yml file from its jinja template
+            render_snowflake_yml(
+                parent_to_snowflake_yml=current_working_directory.joinpath(project_name)
+            )
+
+            # Render README.md file from its jinja template
+            render_nativeapp_readme(
+                parent_to_readme=current_working_directory.joinpath(
+                    project_name, "app"
+                ),
+                project_name=project_name,
+            )
```

Comment:
```suggestion
            project_path = current_working_directory / project_name
            render_snowflake_yml(
                parent_to_snowflake_yml=project_path
            )

            # Render README.md file from its jinja template
            render_nativeapp_readme(
                parent_to_readme=project_path / "app",
                project_name=project_name,
            )
```

Using `/` operator is good practice and makes the code more intuitive in case of paths